Application.py

```diff
@@ -0,0 +1,89 @@
+import Utils
```

Comment:
Maybe it is worth to have a look to pep8 convention and snakecase:
https://realpython.com/python-pep8/
https://khalilstemmler.com/blogs/camel-case-snake-case-pascal-case/
(Anyway looks good)